src/CommunityToolkit/Xamarin.CommunityToolkit/Startup/ToolkitPlatform.android.cs

```diff
@@ -6,12 +6,12 @@ namespace Xamarin.CommunityToolkit
 	/// <summary>
 	/// Platform extension methods.
 	/// </summary>
-	internal static class ToolkitPlatform
+	static class ToolkitPlatform
 	{
 		/// <summary>
 		/// Gets the <see cref="Context"/>.
 		/// </summary>
-		internal static Context Context
+		internal static Context? Context
```

Comment:
This can't be null.

Reply:
<img width="897" alt="image" src="https://user-images.githubusercontent.com/13558917/110228899-bf644080-7eb9-11eb-8f52-69f2188e4444.png">

The Xamarin.Android SDK has marked `View.Context` as nullable, so we have to propagate it upwards.

If we don't want `ToolkitPlatform.Context` to be nullable, we could do this as an alternative: 

```csharp
internal static Context Context
{
	get
	{
		var page = Forms.Application.Current.MainPage;
		var renderer = page.GetRenderer();
		return renderer.View.Context ?? throw new NullReferenceException();
	}
}
```